src/main/java/com/amadeus/travel/TripParser.java

```diff
@@ -89,7 +89,7 @@ public TripDetail post(String body) throws ResponseException {
    */
   public TripDetail post(File file) throws ResponseException, IOException {
```

Comment:
But I don't how to use this method even for the original one. I got a long string from the [test case](https://developers.amadeus.com/self-service/category/trip/api-doc/trip-parser/api-reference), I tried this string on the ```post(string)``` and it works. Then I copy this string and paste it into a pdf file, then try ```post(file)``` but it fails with "com.amadeus.exceptions.ServerException: [500]
Unfortunately your request could not be processed. We are enhancing our system to support this request in future."